src/app/public/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -423,7 +438,7 @@ export class SkyDatepickerInputDirective
     if (value instanceof Date) {
       dateValue = value;
     } else if (typeof value === 'string') {
-      const date = this.dateFormatter.getDateFromString(value, this.dateFormat);
+      const date = this.dateFormatter.getDateFromString(value, this.dateFormat, this.strictFormat);
```

Comment:
Maybe just name the input property `strict`? Since we have a `dateFormat` input, it might be confusing to have another input incorporate the word "format".

"Are you asking for a date format and a _strict_ date format?"

Reply:
Good feedback. I like `strict` better too. Changed.